packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -79,14 +79,10 @@ function ClickAwayListener(props) {
     if (event.composedPath) {
       insideDOM = event.composedPath().indexOf(nodeRef.current) > -1;
     } else {
-      const doc = ownerDocument(nodeRef.current);
       // TODO v6 remove dead logic https://caniuse.com/#search=composedPath.
-      // `doc.contains` works in modern browsers but isn't supported in IE 11:
```

Comment:
I guess there's no point mentioning `doc.contains` at all because this branch will be replaced entirely with `event.composedPath` in v6